village/src/components/SmurfForm.js

```diff
@@ -13,6 +14,11 @@ class SmurfForm extends Component {
   addSmurf = event => {
     event.preventDefault();
     // add code to create the smurf using the api
+    axios.post('http://localhost:3333/smurfs', {name: this.state.name,
+                                                age: this.state.age,
+                                                height: this.state.height})
+          .then(window.location.reload())
```

Comment:
not an ideal approach, it would be preferred to pass up it up to App using a method on props. It works for MVP but I would recommend figuring out a different approach.

Also, your `.then` needs to take a callback, you're giving it a function call, not ideal either